tests/manage/test_create_storage_class_pvc.py

```diff
@@ -1,135 +1,139 @@
-        condition='Running', selector='app=rook-ceph-mds'
+    global RBD_PVC_OBJ, RBD_POD_OBJ, rbd_pv
+    log.info('deleting rbd pod')
+    RBD_POD_OBJ.delete()
+    log.info("Deleting RBD PVC")
+    RBD_PVC_OBJ.delete()
+    assert helpers.validate_pv_delete(rbd_pv)
+    log.info("Deleting CEPH BLOCK POOL")
+    RBD_POOL.delete()
+    log.info("Deleting RBD Secret")
+    RBD_SECRET_OBJ.delete()
+    log.info("Deleting RBD Storageclass")
+    RBD_SC_OBJ.delete()
+
+
+@pytest.fixture(scope='function')
+def test_fixture_cephfs(request):
+    def finalizer():
+        teardown_fs()
+
+    request.addfinalizer(finalizer)
+    setup_fs()
+
+
+def setup_fs():
+    log.info("Creating CEPHFS Secret")
+    global CEPHFS_SECRET_OBJ
+    CEPHFS_SECRET_OBJ = helpers.create_secret(constants.CEPHFILESYSTEM)
+
+    global CEPHFS_SC_OBJ
+    log.info("Creating CephFS Storage class ")
+    CEPHFS_SC_OBJ = helpers.create_storage_class(
+        constants.CEPHFILESYSTEM,
+        helpers.get_cephfs_data_pool_name(),
+        CEPHFS_SECRET_OBJ.name
     )
-    pods = POD.get(selector='app=rook-ceph-mds')['items']
-    assert len(pods) == 2
 
 
 def teardown_fs():
-    """
-    Tearing down the environment
-    """
-    global CEPH_OBJ
-    CEPH_OBJ.delete()
+    global CEPHFS_PVC_OBJ, CEPHFS_POD_OBJ, cephfs_pv
+    log.info('deleting cephfs pod')
+    CEPHFS_POD_OBJ.delete()
+    log.info('deleting cephfs pvc')
+    CEPHFS_PVC_OBJ.delete()
+    assert helpers.validate_pv_delete(cephfs_pv)
+    log.info("Deleting CEPHFS Secret")
+    CEPHFS_SECRET_OBJ.delete()
+    log.info("Deleting CephFS Storageclass")
+    CEPHFS_SC_OBJ.delete()
 
 
 @tier1
 class TestOSCBasics(ManageTest):
-    mons = (
-        f'rook-ceph-mon-a.{config.ENV_DATA["cluster_namespace"]}'
-        f'.svc.cluster.local:6789,'
-        f'rook-ceph-mon-b.{config.ENV_DATA["cluster_namespace"]}.'
-        f'svc.cluster.local:6789,'
-        f'rook-ceph-mon-c.{config.ENV_DATA["cluster_namespace"]}'
-        f'.svc.cluster.local:6789'
-    )
-
     @pytest.mark.polarion_id("OCS-336")
-    def test_basics_rbd(self, test_fixture):
+    def test_basics_rbd(self, test_fixture_rbd):
         """
         Testing basics: secret creation,
-        storage class creation and pvc with cephfs
+        storage class creation,pvc and pod with rbd
         """
-        self.cephfs_secret = templating.load_yaml_to_dict(
-            constants.CSI_CEPHFS_SECRET_YAML
-        )
-        del self.cephfs_secret['data']['userID']
-        del self.cephfs_secret['data']['userKey']
-        self.cephfs_secret['data']['adminKey'] = (
-            get_admin_key_from_ceph_tools()
-        )
-        self.cephfs_secret['data']['adminID'] = constants.ADMIN_BASE64
-        logging.info(self.cephfs_secret)
-        secret = OCS(**self.cephfs_secret)
-        secret.create()
-        self.cephfs_sc = templating.load_yaml_to_dict(
-            constants.CSI_CEPHFS_STORAGECLASS_YAML
+        global RBD_PVC_OBJ, RBD_POD_OBJ, rbd_pv
```

Comment:
rbd_pv is global?
why not in a capital letters

Reply:
Made in capital letters , fixed